NGitLab.Mock/Repository.cs

```diff
@@ -323,6 +323,11 @@ public IEnumerable<Commit> GetBranchCommits(string branchName)
             }
         }
 
+        public IEnumerable<Commit> GetCommits()
+        {
+            return GetCommits(string.Empty);
```

Comment:
```suggestion
            return GetCommits(null);
```

Reply:
I can't make it null, it would be ambiguous which overload I'm trying to call.